graphql.schema.json

```diff
@@ -6403,6 +6403,38 @@
                 "ofType": null
               }
             },
+            "isDeprecated": true,
+            "deprecationReason": "Use `selectedSortColumn` and `selectedSortDirection`"
+          },
+          {
+            "name": "selectedSortColumn",
```

Comment:
this is a little tricky, multiple changes in `ti` repo related to helium graphql are still in-flight. i did an introspection of the schema against my local ti instance with partial changes, this causes the graphql document updates to undo some changes previously done.